### PR TITLE
Update feature request template

### PR DESCRIPTION
**Pull request by sharmasejal**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+labels: enhancement
+assignees: ''
+
+---
+
+------------------------------- delete below -------------------------------
+
+Thank you for raising a feature request! To get started, here's a few things to consider:
+
+# 1) Check existing requests:
```
Comment (drwill-ms) [inline .github/ISSUE_TEMPLATE/feature_request.md:14]: minor: use `#1` or `1)`

Don't mix the styles. :)

Comment (sharmasejal) [reply .github/ISSUE_TEMPLATE/feature_request.md:14]: Tried removing '#' but then figured out that it is a style marker to set the contents of that line to bold headline. Therefore, added it back.

Comment (drwill-ms) [reply .github/ISSUE_TEMPLATE/feature_request.md:14]: Ah, I see. It is markdown.

**Pull request merged**